x-pack/plugins/license_management/public/components/telemetry_opt_in/telemetry_opt_in.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+import {
+  EuiLink,
+  EuiCheckbox,
+  EuiSpacer,
+  EuiText,
+  EuiPopover
+} from '@elastic/eui';
+import { showTelemetryOptIn, getTelemetryFetcher, PRIVACY_STATEMENT_URL, OptInExampleFlyout } from '../../lib/telemetry';
+
+export class TelemetryOptIn extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      showMoreTelemetryInfo: false,
+      isOptingInToTelemetry: false,
+      showExample: false
+    };
+  }
+  isOptingInToTelemetry = () => {
+    return this.state.isOptingInToTelemetry;
+  }
+  closeReadMorePopover = () => {
+    this.setState({ showMoreTelemetryInfo: false });
+  }
+  render() {
+    const { showMoreTelemetryInfo, isOptingInToTelemetry, showExample } = this.state;
+    const { isStartTrial } = this.props;
+
+    let example = null;
+    if (showExample) {
+      example = (
+        <OptInExampleFlyout
+          onClose={() => this.setState({ showExample: false })}
+          fetchTelemetry={getTelemetryFetcher}
+        />
+      );
+    }
+
+    let toCurrentCustomers;
+    if (!isStartTrial) {
+      toCurrentCustomers = (
+        <Fragment>
+          <EuiText>
+            <p>Gold and platinum customers: help support give you better service.</p>
+          </EuiText>
+          <EuiSpacer  size="s"/>
+        </Fragment>
+      );
+    }
+
+    const readMoreButton = (
+      <EuiLink onClick={() => { this.setState({ showMoreTelemetryInfo: !showMoreTelemetryInfo });}}>
+      Read more
+      </EuiLink>
+    );
+
+    const popover = (
+      <EuiPopover
+        ownFocus
+        id="readMorePopover"
+        button={readMoreButton}
+        isOpen={showMoreTelemetryInfo}
+        closePopover={this.closeReadMorePopover}
+        style={{ verticalAlign: 'baseline' }}
+      >
+        <EuiText style={{ width: 240 }} >
```

Comment:
This is how we've been handling sizing the width's of popovers ([see first example](https://elastic.github.io/eui/#/layout/popover)). Just usually it's a div that contains the content, I just skipped the extra div and put it directly on the text container.

Reply:
I think originally we were using styles in the EUI examples to make it clearer that custom styles were in use (otherwise it would seem like the classes were something provided by EUI).

Reply:
done